app.js

```diff
@@ -1,72 +1,99 @@
-const formContainer = document.getElementById('formContainer');
+/* eslint-disable max-classes-per-file */
+
+// const formContainer = document.getElementById('formContainer');
```

Comment:
- [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.